frontend/awx/access/organizations/OrganizationPage/OrganizationPage.tsx

```diff
@@ -31,12 +31,12 @@ export function OrganizationPage() {
     data: organization,
     error,
     refresh,
-  } = useGet<Organization>(`/api/v2/organizations/${params.id ?? ''}/`);
-  const history = useNavigate();
+  } = useGetItem<Organization>('/api/v2/organizations/', params.id);
+  const nagivate = useNavigate();
```

Comment:
should be 'navigate'

Reply:
thats what it was changed to

Reply:
@jamestalton `useNavigate` hook is set to the variable that is misspelled  as`nagivate`